### PR TITLE
FPK-279: Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by donovanroubos**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Documentation
+
+If a PR introduces or changes API, mention one of the Yummygum team members to make sure it'll be processed into the [website docs](https://flagpack.xyz/docs/).
+
+## Development
+
+Follow these steps to get your code PR-ready:
+
+1. Fork the `react-flagpack` repo (and also the `flagpack-core` if you want to make changes to the core)
```
Comment (daoneandonly) [inline CONTRIBUTING.md:19]: Let's, somehwere in this document, reference the `flagpack-core` repo with a link.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Bug fixes
+
+Issues can be found on our [GitHub issues](https://github.com/yummygum/react-flagpack/issues) page. Found a bug? Let us know! Report the bug by [creating a new issue](https://github.com/Yummygum/react-flagpack/issues/new/choose). Make sure to check the issues page before creating a new bug issue to prevent duplicate issues. Once you start fixing a bug, post your intent on the issue to prevent others diving in at the same time.
```
Comment (daoneandonly) [inline CONTRIBUTING.md:5]: Do we have a system system in place regarding issue tags? For instance `bug` or `feature-request`. Whether we're using the standard Github tags or using our own, we should mention and explain how people can use these tags.

**Review by daoneandonly — CHANGES_REQUESTED** [review]
Looking forward to some contributions. However I have a few minor things I'd like to see added.

File: CONTRIBUTING.md (hunk 2 above)
Comment (donovanroubos) [reply CONTRIBUTING.md:5]: There are two different types of issues that people can make (Bug report and Feature request). Also the labels will explain which one they need to use. I think it's very self-explaining.

image

**Review by daoneandonly — APPROVED** [review]

**Pull request merged**